CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,8 @@
+# Code of Conduct
```

Comment:
I'm not sure that this fully satisfies the requirements. From the link you provided, it states that you should, "Place a CODE_OF_CONDUCT file in your project’s root directory, and link to it from your README, so it’s visible to your community."

You created the file, but readme.md doesn't link to the file.